packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,14 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement.
+class Announcement {
```

Comment:
Let's rename this to `CapturedAccessibilityAnnouncement`. Otherwise it sounds too much like a production class name. Also, "announcement" does not by itself imply accessibility.

Reply:
The only potential issue would be lines getting longer than 80 character. I could always break lines similar to what dart formatter would do though.

Reply:
Done!